### PR TITLE
feat(TMRX-1590): refactor lead story slices

### PR DESCRIPTION
**Pull request by douglasmik**
### Description

Refactor of Lead slices to remove breakpoint loops
[TMRX-1590](https://nidigitalsolutions.jira.com/browse/TMRX-1590)

### Checklist

- [ ] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):

Include screenshots if needed.

[TMRX-1590]: https://nidigitalsolutions.jira.com/browse/TMRX-1590?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/slices/shared/layouts.tsx [hunk 1]
```diff
@@ -30,6 +29,8 @@ export const CustomStackLayout: React.FC<CustomStackLayoutProps> = ({
         md: 'space000'
       }}
       marginBlockEnd={marginBlockEnd || 'space060'}
+      // @ts-ignore
+      className={className}
```
Comment (milanharia-tm) [inline packages/ts-newskit/src/slices/shared/layouts.tsx:33]: Why do we need ts-ignore?

File: packages/ts-newskit/src/slices/shared-styles/index.ts [hunk 2]
```diff
@@ -7,6 +7,54 @@ import {
...
+const setMarginBlockStart = (space: string) => ({
+  marginBlockStart: `${space}`
+});
+export const LeadStoryContainer = styled(CustomStackLayout)<{
+  className?: string;
+}>`
+  ${getMediaQueryFromTheme('xs', 'lg')} {
+    hr[aria-label='article-divider-horizontal'] {
+      display: none;
+    }
```
Comment (milanharia-tm) [inline packages/ts-newskit/src/slices/shared-styles/index.ts:21]: This style causes dividers to disappear from `ArticleStackSmall` in mobile on lead story 1

Screenshot 2023-10-20 at 12 37 00

File: packages/ts-newskit/src/slices/shared/layouts.tsx (hunk 1 above)
Comment (douglasmik) [reply packages/ts-newskit/src/slices/shared/layouts.tsx:33]: the `Stack` Newskit component doesn't accept `className` by default but is required to extend styles. Making this a styled component with `className` causes other TS errors also, so adding it here is the only solution until we get a version bump in Newskit to accept this as a prop as some of the other components do

File: packages/ts-newskit/src/slices/shared-styles/index.ts (hunk 2 above)
Comment (douglasmik) [reply packages/ts-newskit/src/slices/shared-styles/index.ts:21]: good spot - updated style

**Review by milanharia-tm — APPROVED** [review]

**Review by ibrahim-kurhan — APPROVED** [review]

**Pull request merged**